migrations/scripts/02-07-2026-normalize-membership-invitation-user-ids.js

```
@@ -0,0 +1,198 @@
+function normalizeUserId(userId) {
+  return String(userId || "").trim().toLowerCase();
```

ESLint

[error] Replace `.trim()` with `⏎····.trim()⏎····`

(prettier/prettier)